fiftyone/migrations/revisions/v0_6_2.py

```diff
@@ -21,20 +35,36 @@ def up(db, dataset_name):
     if "media_type" not in dataset_dict:
         dataset_dict["media_type"] = "image"
 
-    for field in dataset_dict["sample_fields"] + dataset_dict["frame_fields"]:
+    if "frame_fields" not in dataset_dict:
+        dataset_dict["frame_fields"] = []
+
+    for field in dataset_dict["sample_fields"]:
         if "media_type" in field:
             del field["media_type"]
         if field["name"] == "frames" and dataset_dict["media_type"] == "video":
             field["ftype"] = "fiftyone.core.fields.EmbeddedDocumentField"
             field["subfield"] = None
             field["embedded_doc_type"] = "fiftyone.core.labels._Frames"
 
+    for field in dataset_dict["frame_fields"]:
+        if "media_type" in field:
+            del field["media_type"]
+
     db.datasets.replace_one(match_d, dataset_dict)
+    sample_coll = dataset_dict["sample_collection_name"]
 
     if dataset_dict["media_type"] == "image":
+        writes = []
+        for s in db[sample_coll].find():
+            converted = False
+            for d in s.items():
+                converted = _up_convert_polyline_points(s)
```

Comment:
This change would catch cases where only some (but not the last) instances are converted:
```suggestion
                converted |= _up_convert_polyline_points(s)
```
Same goes for the other two instances of loops like this.